core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Database.java

```diff
@@ -0,0 +1,78 @@
+package com.sequenceiq.cloudbreak.domain.stack;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.database.DatabaseAvailabilityType;
+import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.common.json.JsonToString;
+import com.sequenceiq.cloudbreak.domain.ProvisionEntity;
+import com.sequenceiq.cloudbreak.domain.converter.DatabaseAvailabilityTypeConverter;
+import com.sequenceiq.cloudbreak.workspace.repository.EntityType;
+
+@Entity
+@EntityType(entityClass = Database.class)
+@Table(name = "database")
+public class Database implements ProvisionEntity {
```

Comment:
is it intentional that we managing this in core and not in redbeams?

Reply:
Unforunatelly we have to store the db related basic parameters in both sdx and core, as we create the database later in a flow step using redbeams api.